datastream-server/src/main/java/com/linkedin/datastream/server/assignment/BroadcastStrategy.java

```diff
@@ -93,6 +107,13 @@ public BroadcastStrategy(Optional<Integer> maxTasks) {
     return newAssignment;
   }
 
+  private DatastreamTask getOrCreateDatastreamTask(List<DatastreamTask> reuseTasksPerDg, DatastreamGroup dg) {
+    if (reuseTasksPerDg.size() > 0) {
+      return reuseTasksPerDg.remove(0);
+    }
+    return new DatastreamTaskImpl(dg.getDatastreams());
```

Comment:
You can modify this to be:

`return ((reuseTasksPerDg.size() > 0) ? reuseTasksPerDg.remove(0) : new DatastreamTaskImpl(dg.getDatastreams()));`

You can also inline this where it is called if you'd like.

Reply:
Done.